Makefile

```diff
@@ -5,30 +5,23 @@ export $(shell sed 's/=.*//' .env)
 test:
 	env
 
+# delete this
 up-prod:
 	sudo -E docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
 	sudo -E docker-compose exec web /bin/sh -c "python3 manage.py migrate"
 	sudo -E docker-compose exec web /bin/sh -c "python3 manage.py compilemessages -l en"
 	sudo -E docker-compose exec web /bin/sh -c "python3 manage.py collectstatic --noinput"
 
+# delete this too
 up-develop:
 	# sleep 5 && sudo docker-compose exec web /bin/sh -c "python3 manage.py migrate" &
 	sudo -E docker-compose run web /bin/sh -c "python3 manage.py compilemessages -l en"
 	sudo -E docker-compose -f docker-compose.yml -f docker-compose.develop.yml up
 
 makemessages:
-	sudo -E docker-compose run web python3 manage.py makemessages -l en
-
-# Use when the container is running
-makemessages-exec:
-	sudo -E docker-compose exec web python3 manage.py makemessages -l en
+	sudo -E docker compose exec web python3 manage.py makemessages -a
+	sudo -E docker compose cp web:/locale .
 
 compilemessages:
-	sudo -E docker-compose exec web python3 manage.py compilemessages -l en
-
-
-build:
-	sudo -E docker-compose build
-
-down:
-	sudo -E docker-compose down
```

Comment:
These two are referred to in the README, so it should be updated with new build instructions.

Reply:
Could also just revert the change

Reply:
Also works